### PR TITLE
Project 1: Personal Blog

### PR DESCRIPTION
**Pull request by seb-lex**
This is the first project from Udacity's Web Dev course: building a personal blog site. The final code:

The link to project rubric:

https://learn-udacity.top/fewd586261/Front%20End%20Web%20Developer%20v5.0.0/Part%2002-Module%2001-Lesson%2006_Personal%20Blog%20Website/Project%20Rubric%20-%20Personal%20Blog%20Website.html

Video demo:

https://user-images.githubusercontent.com/106308738/176857490-74ee370e-4bbf-4440-8b35-95bd2b7b13fa.mp4

### CODE REVIEW

File: Part 1 - Personal Blog/style.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@700&family=Roboto&display=swap');
```
Comment (Martian2Lee) [inline Part 1 - Personal Blog/style.css]: 👍

File: Part 1 - Personal Blog/style.css [hunk 2]
```diff
@@ -95,3 +95,37 @@ ul {
 .header img {
     margin-left: 10px;
 }
+
+/* content area */
```
Comment (Martian2Lee) [inline Part 1 - Personal Blog/style.css:101]: in big files, comments to separate the code is very important 👍

File: Part 1 - Personal Blog/style.css [hunk 3]
```diff
@@ -129,3 +134,26 @@ button {
     padding: 3px;
     margin-top: 5px;
 }
+
+/* profile */
+.profile {
+    background-color: darkblue;
+    /* max-height: 200px; */
```
Comment (Martian2Lee) [inline Part 1 - Personal Blog/style.css]: you can add a comment `// TODO` reminding you to clean this up later

File: Part 1 - Personal Blog/style.css [hunk 4]
```diff
@@ -157,3 +157,23 @@ button {
     justify-content: space-evenly;
     margin-top: 10px;
 }
+
+/* responsive */
+
```
Comment (Martian2Lee) [inline Part 1 - Personal Blog/style.css:172]: please remove this empty line 162

**Review by Martian2Lee — CHANGES_REQUESTED** [review]

File: Part 1 - Personal Blog/style.css [hunk 5]
```diff
@@ -0,0 +1,179 @@
+/* base styles */
+
```
Comment (Martian2Lee) [inline Part 1 - Personal Blog/style.css]: please remove this empty line 2

**Review by Martian2Lee — CHANGES_REQUESTED** [review]

**Comment by sebz996** [comment]
Done.

https://codepen.io/sebz996/pen/KKowvOE

File: Part 1 - Personal Blog/style.css (hunk 5 above)
Comment (seb-lex) [reply Part 1 - Personal Blog/style.css]: Line removed.

File: Part 1 - Personal Blog/style.css (hunk 4 above)
Comment (seb-lex) [reply Part 1 - Personal Blog/style.css:172]: Line removed. https://github.com/sebz996/udacity-front-end-web-dev-2/pull/1/commits/0563ee40a5ff1c40e17d1bea5b55630147a43a5f

**Review by leemoispace — COMMENTED** [review]
my first review test.

File: Part 1 - Personal Blog/index.html [hunk 6]
```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/7dab968d48.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Seb Lex's Personal Blog</title>
```
Comment (leemoispace) [inline Part 1 - Personal Blog/index.html:10]: review test, good job Seb!

**Review by leemoispace — APPROVED** [review]
with a review in code.

File: Part 1 - Personal Blog/style.css [hunk 7]
```diff
@@ -0,0 +1,199 @@
+/* base styles */
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400&family=Roboto&display=swap');
+
```
Comment (leemoispace) [inline Part 1 - Personal Blog/style.css:5]: structure-wise, Rubrics asked:
1. There should be at least 3 imported files in the main CSS file.
2. Files are organized with a directory structure that separates files based on page and functionality.

File: Part 1 - Personal Blog/post.html [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+
+
+    <div class="content container">
+        <div class="cards">
+            <article class="card">
+                <h2>Article 1</h2>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Laboriosam ullam natus culpa harum, fugit
+                    reiciendis similique! Quaerat nostrum, ipsum aspernatur id vel deleniti officia consequatur
+                    consectetur qui sequi eius quia.</p>
+
```
Comment (leemoispace) [inline Part 1 - Personal Blog/post.html:35]: Rubric ask to add typography selector in article.

File: Part 1 - Personal Blog/post.html [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+                </ul>
+            </div>
+        </aside>
+    </div>
+
+
+
+
+</body>
+
```
Comment (leemoispace) [inline Part 1 - Personal Blog/post.html:74]: footer required.

**Review by leemoispace — COMMENTED** [review]
checked by rubrics standard and comment in file.

File: Part 1 - Personal Blog/post.html (hunk 9 above)
Comment (seb-lex) [reply Part 1 - Personal Blog/post.html:74]: Done.

File: Part 1 - Personal Blog/style.css (hunk 7 above)
Comment (seb-lex) [reply Part 1 - Personal Blog/style.css:5]: Done.

File: Part 1 - Personal Blog/post.html (hunk 8 above)
Comment (seb-lex) [reply Part 1 - Personal Blog/post.html:35]: Done.

**Review by leemoispace — APPROVED** [review]
Good job, Seb, keep it up!

**Review by Martian2Lee — APPROVED** [review]
🚀

**Pull request merged**